packages/itwinui-react/src/core/StatusMessage/StatusMessage.tsx

```diff
@@ -51,14 +53,17 @@ export const StatusMessage = React.forwardRef((props, ref) => {
 
   const icon = userStartIcon ?? (status && StatusIconMap[status]());
 
+  // If user passes null, we don't want to show the icon
+  const shouldShowIcon = userStartIcon !== null && !!icon;
```

Comment:
This change made me realize something: we can rely on the default icon behavior of `StatusMessage` in more places. 

Can we go through all uses of `StatusMessage` and remove the ones where we are unnecessarily passing a `startIcon` from `StatusIconMap`?

Reply:
Sounds good, done (https://github.com/iTwin/iTwinUI/pull/1753/commits/19e955ce6cfd45db7040dd7a7c79e32f1ad0ae33).

When doing this for `InputGroup`, I noticed some potentially incorrect behavior of the icon in the main branch: when adding a status (with or without message), does it feel like a third radio button exists and has been checked?
 
<img width="193" alt="Screenshot 2024-01-12 at 9 51 19 AM" src="https://github.com/iTwin/iTwinUI/assets/45748283/e98f7c60-9575-48cb-ac6a-b6a8fe1a05ac">

I created a [Stackblitz playground](https://stackblitz.com/edit/github-3frwv5?file=src%2FApp.tsx) for this.

Reply:
Can't you just pass `svgIcon` directly to the `StatusMessage`? Instead of creating an `icon()` wrapper

And the `InputGroup` behavior is not incorrect. That's just how the icon looks. https://itwin.github.io/iTwinUI-icons/#icons-status-success

Reply:
I had kept the `icon()` wrapper to maintain the logic of showing a status icon only when `status && message`.

`LabeledSelect`:
But looks like that is handled by the check `if (typeof message === "string")`:

```jsx
{typeof message === 'string' ? (
  <StatusMessage
    status={status}
    startIcon={displayStyle === 'default' ? icon() : undefined}
  >
    {message}
  </StatusMessage>
) : (
  message
)}
```

So, when message is `undefined`, we fall into the else branch and render `undefined` (`message`) instead of `StatusMessage`. So yes, looks like the `icon()` wrapper can be removed from `LabeledSelect`.

---

`InputGroup`:
Upon re-reading the previous logic in `icon()` and the `return`, looks like the previous logic can still be maintained by removing the `icon()` wrapper.

---

So removed in both places (https://github.com/iTwin/iTwinUI/pull/1753/commits/5f053f94d85f3a83e3a55841958d88660d2115f9)

Also, I noticed that the `displayStyle === 'default' ? icon() : undefined` check in `LabeledSelect` was not really doing anything ([StackBlitz playground](https://stackblitz.com/edit/github-ejxhhk?file=src%2FApp.tsx)). So I changed it to directly be `svgIcon`.